modules/round/src/main/RoundSocket.scala

```diff
@@ -72,11 +72,12 @@ private[round] final class RoundSocket(
       lilaBus.ask[Set[User.ID]]('simulGetHosts)(GetHostIds).map(_ contains u)
     }
 
-    def weightedRagequitTimeout = ragequitTimeout.toMillis * goneWeight
-    def weightedDisconnectTimeout = gameDisconnectTimeout(gameSpeed).toMillis * goneWeight
+    private def timeoutMillis = {
+      if (isBye) ragequitTimeout.toMillis else gameDisconnectTimeout(gameSpeed).toMillis
+    } * goneWeight atLeast 12000
```

Comment:
You probably did it correctly but confirming this can't hurt: is the operator precedence of atLeast lower than multiplication?